crates/cli/src/args/encrypt.rs

```diff
@@ -71,10 +71,22 @@ mod mock {
 
 #[cfg(test)]
 mod test {
+    use regex::Regex;
     use rops::{file::metadata::PartialEncryptionConfig, test_utils::*};
 
     use super::*;
 
+    #[test]
+    fn merges_integration_keys_from_config_stdin() {
```

Comment:
Test looks good! I would like the name of this test to better reflect the issue it tries to prevent regressions against. 
Maybe `merges_configuration_for_stdin_with_fallback_regex`?